src/index.js

```diff
@@ -5,26 +5,70 @@ import Traveler from './Traveler';
 import Destination from "./Destination";
 import Trip from './Trip';
 
-let traveler = new Traveler();
-let trip = new Trip()
-let destination = new Destination()
-let allDestinationsData = [];
-let allTripsData = [];
+let currentTraveler;
+let currentTravelerTrips;
+let currentTravelerDestinations
+let allDestinationsData;
+let allTripsData;
 
 window.onload = generateAPIData()
 
-// function generateSingleTravelerAPI(id) {
-//   fetchData
-// }
+const loginButton = document.getElementById('login-submit');
+const logOutButton = document.getElementById('log-out-button');
+const loginView = document.querySelector('.login-view');
+const userView = document.querySelector('.traveler-view');
+
+loginButton.addEventListener('click', validateLoginForm);
+logOutButton.addEventListener('click', logOut);
```

Comment:
Eventually it might be helpful to put all your click handlers into one helper function & use conditional logic to make sure the right function happens